From db01a787563510443db1e8f5feba8f405712f9e7 Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Mon, 1 May 2017 08:52:48 -0500
Subject: [PATCH] 0001-make-bacon

---
 core/Makefile | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index d804c37..0de4de0 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1849,8 +1849,14 @@ $(INTERNAL_OTA_PACKAGE_TARGET): $(BUILT_TARGET_FILES_PACKAGE)
 	   $(if $(OEM_OTA_CONFIG), -o $(OEM_OTA_CONFIG)) \
 	   $(BUILT_TARGET_FILES_PACKAGE) $@
 
-.PHONY: otapackage
+AOSP_VERSION := $(TARGET_PRODUCT)-$(PLATFORM_VERSION)-$(BUILD_ID)-$(shell date -u +%Y%m%d)-$(USER)
+AOSP_TARGET_PACKAGE := $(PRODUCT_OUT)/$(AOSP_VERSION).zip
+
+.PHONY: otapackage bacon
 otapackage: $(INTERNAL_OTA_PACKAGE_TARGET)
+bacon: otapackage
+	$(hide) ln -f $(INTERNAL_OTA_PACKAGE_TARGET) $(AOSP_TARGET_PACKAGE)
+	@echo -e ${CL_CYN}"Package Complete: $(AOSP_TARGET_PACKAGE)"${CL_RST}
 
 endif    # build_ota_package
 
-- 
1.9.1

